confgen/sample-proxyfs-configuration/proxyfs-shares.conf

```diff
@@ -7,12 +7,12 @@ ReadCacheLineSize: 1048576
 ReadCacheWeight:   100
```

Comment:
I noticed that there is a file **pfs.txt** in confgen/sample-proxyfs-configuration.    I think we should remove that file since it appears to be out of date.

Reply:
Done.